cmake/dependencies.cmake

```diff
@@ -151,6 +151,9 @@ function(ov_download_tbb)
     if(EXISTS "${TBBROOT}/lib/cmake/TBB/TBBConfig.cmake")
         # oneTBB case
         update_deps_cache(TBB_DIR "${TBBROOT}/lib/cmake/TBB" "Path to TBB cmake folder")
+    elseif(EXISTS "${TBBROOT}/lib/cmake/tbb/TBBConfig.cmake")
```

Comment:
on Windows, I have `TBB`. On which OS do you observe `tbb`?

Reply:
on Linux, oneTBB version less than 2021.6.0, the TBBConfig.cmake path is "oneapi-tbb-2021.2.0/lib/cmake/tbb/"

Reply:
In `tbb` is right path, you also need to update the following places as well:
```
C:\Users\ilavreno\Documents\Programming\git_repo\openvino\cmake\templates\OpenVINOConfig.cmake.in:
  160                  # oneTBB case exposed via export TBBROOT=<custom TBB root>
  161                  "${ENV_TBBROOT}/lib64/cmake/TBB"
  162:                 "${ENV_TBBROOT}/lib/cmake/TBB"
  163                  # "$ENV{TBB_DIR}"
  164                  # for custom TBB exposed via cmake -DTBBROOT=<custom TBB root>

C:\Users\ilavreno\Documents\Programming\git_repo\openvino\scripts\setupvars\setupvars.sh:
   54      export LD_LIBRARY_PATH=$INSTALLDIR/runtime/3rdparty/tbb/lib:${LD_LIBRARY_PATH:+$LD_LIBRARY_PATH}
   55  
   56:     if [ -e "$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/TBB" ]; then
   57:         export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/lib/cmake/TBB
   58      elif [ -e "$INSTALLDIR/runtime/3rdparty/tbb/lib64/cmake/TBB" ]; then
   59          export TBB_DIR=$INSTALLDIR/runtime/3rdparty/tbb/lib64/cmake/TBB
```

Reply:
@sunxiaoxia2022 
I have tried 2021.5 and I see `TBB`:
```
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/cmake/TBB/TBBTargets.cmake
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/cmake/TBB/TBBTargets-relwithdebinfo.cmake
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/cmake/TBB/TBBConfig.cmake
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/cmake/TBB/TBBConfigVersion.cmake
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/share/doc/TBB/README.md
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbb.so.12.5
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbb.so.12
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbb.so
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/pkgconfig/tbb.pc
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc.so.2.5
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc.so.2
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc.so
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc_proxy.so.2.5
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc_proxy.so.2
-- Set runtime path of "/home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc_proxy.so.2.5" to ""
-- Installing: /home/ilyal/Documents/Programming/build/onetbb-release/install/lib/libtbbmalloc_proxy.so
(py39) ilyal@wanglei-nuc:~/Documents/Programming/build/onetbb-release$ cd ../../git_repo/oneTBB/
(py39) ilyal@wanglei-nuc:~/Documents/Programming/git_repo/oneTBB$ git branch
* (HEAD detached at v2021.5.0)
* 
```

Reply:
It seems that if we build oneTBB from source code, the install path use "TBB" , but if we use oneTBB in oneapi release package, it will use "tbb".

Reply:
ok, let's update the places mentioned above as well

Reply:
added "tbb" to OpenVINOConfig.cmake.in and setupvars.sh.